charts/trino/templates/secret.yaml

```
@@ -1,15 +1,26 @@
-{{- if or .Values.auth.passwordAuth .Values.auth.groups }}
+{{-  if and (not .Values.auth.passwordAuthSecret) (.Values.auth.passwordAuth) }}
```

Remove the extra space and redundant parenthesis:
```suggestion
{{- if and (not .Values.auth.passwordAuthSecret) .Values.auth.passwordAuth }}
```

---

done